Log a final, machine readable status

rhodesn opened the pull request:
Resolves #213 

Log to /var/lib/auter/last-{prep,apply}-output-X with either:
```
STATUS:SUCCESS:Message
-or-
STATUS:FAILED:Message
```

rhodesn committed:
Log a final, machine readable, status update to
/var/lib/auter/last-{prep,apply}-output-X with either:

STATUS:SUCCESS:Message
-or-
STATUS:FAILED:Message

rhodesn committed:
Write APPLYOUTPUT to last-apply rather than echo'ing directly into the
file

mark-hyde commented on auter.yumdnfModule:
```diff
@@ -164,9 +169,13 @@ function apply_updates() {
+    fi
     default_signal_handling
-    echo -e "$APPLYOUTPUT" &>"${DATADIR}/last-apply-output-${CONFIGSET}"
+    echo -e "$APPLYOUTPUT" > "${DATADIR}/last-apply-output-${CONFIGSET}"
```
Wondering why the change from `&>` to `>` ?

rhodesn replied:
I couldn't work out why we would need stderr from an echo. Have I missed something?

mark-hyde replied:
Good point!
I was thinking it was from the output of the commands themselves! 
Approved then!! :)

rhodesn replied:
Awesome, thanks for pulling it up! I'll look at getting it merged and into a release.

mark-hyde reviewed (approved):
Looks good to me.
Wondering why you went from `&>` to `>` in one of the lines?

rhodesn committed:
Capture errors during the check-update just in case we are running
PACKAGEMANAGEROPTIONS that are not valid and cause the apply phase to
end here.

rhodesn committed:
Update the auter.aptModule to match changes made to auter.yumdnfModule
